### PR TITLE
Highlight int nature of variable

### PR DESCRIPTION
**Pull request by aliphys**

### CODE REVIEW

**Review by per1234 — CHANGES_REQUESTED** [review]
Please follow the established format for documenting return types. You can use this page as a reference:

https://github.com/arduino/reference-en/blob/1625ca154ef7a2108b5fe61e5ab8de4cd122fab7/Language/Functions/Math/sq.adoc?plain=1#L36

File: Language/Functions/Math/map.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ The `map()` function uses integer math so will not generate fractions, when the
 
 [float]
 === Returns
-The mapped value.
+The mapped value. Data type: `int`.
```
Comment (per1234) [inline Language/Functions/Math/map.adoc:54]:
```suggestion
The mapped value. Data type: `long`.
```

Confirm accuracy of information via the function's source code:

https://github.com/arduino/ArduinoCore-API/blob/1.3.1/api/Common.h#L167

**Review by per1234 — CHANGES_REQUESTED** [review]

File: Language/Functions/Math/map.adoc (hunk 1 above)
Comment (aliphys) [reply Language/Functions/Math/map.adoc:54]: As I understand from the linked line you shared @per1234 , the `map()` function uses integer math, with input as the `long` datatype (not `int`). Is this correct?

Comment (per1234) [reply Language/Functions/Math/map.adoc]: >with input as the `long` datatype

It is true that the data type of the parameters (the function's input) is `long`.

However, your proposal here is documenting the return type (the function's output). So the parameter types are not relevant here. The relevant part of the function signature is the `long` that comes before the function name:

```cpp
long map(long, long, long, long, long);
```

That is the return type.

There is a nice tutorial about functions here:

https://cplusplus.com/doc/tutorial/functions/

**Review by per1234 — APPROVED** [review]

**Pull request merged**